src/sentry/models/releaseactivity.py

```diff
@@ -0,0 +1,33 @@
+from django.contrib.postgres.fields import JSONField
+from django.db import models
+from django.utils import timezone
+
+from sentry.db.models import BoundedPositiveIntegerField, FlexibleForeignKey, Model
+
+
+class ReleaseActivity(Model):
+    __include_in_export__ = False
```

Comment:
Tried seeing what this attr is for but to no avail? What is this flag for?

Reply:
It's used for exports here https://github.com/getsentry/sentry/blob/531e83e763186d14c8202190c6fee454e5d0edc4/src/sentry/runner/commands/backup.py#L138. Usually you can leave this as False.